clientgenv2/source_generator.go

```diff
@@ -148,7 +148,7 @@ func (r *SourceGenerator) NewResponseField(selection ast.Selection) *ResponseFie
 		case fieldsResponseFields.IsFragment():
 			// 子フィールドがFragmentの場合はこのFragmentがフィールドの型になる
 			// if a child field is fragment, this field type became fragment.
-			baseType = fieldsResponseFields[0].Type
+			baseType = types.NewPointer(fieldsResponseFields[0].Type)
```

Comment:
will this skip adding a second indirect if the fragment is directly on a `non-!` field?